doc/user_guide/customization.rst

```diff
@@ -710,6 +710,9 @@ outside the chart itself; For example, the container may be a ``<div>`` element
 
 Chart Themes
 ------------
+..
+    _comment: First mention of alt.themes
+
```

Comment:
# TODO
- Replace all `alt.themes` references
- Plan new content
- Add a callout stating when and why the change was made
- Possibly link to related issues